Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/SlidesLimitSwitchTest.java

```diff
@@ -0,0 +1,39 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseTeleOp;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+import java.util.Random;
+
+@TeleOp(name = "SlidesLimitSwitchTest")
+public class SlidesLimitSwitchTest extends BaseTeleOp {
+        Random rng = new Random();
+        @Override
+        public void runOpMode() {
+            initialize();
+            driveSlides(0);
+            waitForStart();
+
+            while (opModeIsActive()) {
+                for(int i = 0; i <= 20; i++) {
+                    for(int j = 0; j <= 5; j++) {
```

Comment:
Can you add a comment explaining what these loops are for?
```suggestion
                for (int i = 0; i <= 20; i++) {
                    for (int j = 0; j <= 5; j++) {
```